Calculator/Calculator/Model/FormManager.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FormManager.swift
+//  Calculator
+//
+//  Created by mint, Whales on 2023/06/14.
+//
+
+import Foundation
+
+enum FormManager {
+    static let numberFormatter = NumberFormatter()
+    
+    static var configuredNumberFormatter : NumberFormatter {
+        self.numberFormatter.numberStyle = .decimal
+        self.numberFormatter.maximumFractionDigits = 19
+        self.numberFormatter.maximumIntegerDigits = 20
+        
+        return self.numberFormatter
+    }
+    
+    static func transformResult(from input: String) -> String {
+        var numberString = deleteComma(input)
+        numberString = deleteLastDot(numberString)
+        
+        guard let number = Double(numberString),
+              let output = self.configuredNumberFormatter.string(from: number as NSNumber) else {
+            return "NaN"
+        }
+        
+        guard output != "-0" else {
+            return "0"
+        }
+        
+        return output
```

Comment:
사바사입니다만, 저는 용법상 if else 문이 자연스러워 보이네요.
```suggestion
        if output == "-0" {
            return "0"
        } else {
            return output
        }
```
선호하지 않는 분들도 많습니다만, 저는 간단한 로직이라면 삼항 연산자도 나쁘지 않다 생각해요.
```swift
return output == "-0" ? "0" : output
```

Reply:
오 삼항연산자 좋은 거 같아요!!!!
삼항연산자로 코드 수정했습니다 감사합니다!!!!